server/ServerApp.cpp

```diff
@@ -260,6 +260,14 @@ void ServerApp::CreateAIClients(const std::vector<PlayerSetupData>& player_setup
         args.push_back(GetOptionsDB().Get<std::string>("log-level"));
     }
 
+    if (GetOptionsDB().Get<bool>("testing")) {
+#ifdef FREEORION_LINUX
+        // Dirty hack to output log to console.
```

Comment:
why not use the log files?

Reply:
I'm not sure if it possible to get them from Travis CI to get detailed information about possible errors.

Reply:
Running the tests isn't possible on AppVeyor as well?

Reply:
It possible but logs will be output to files.